test/gemm/gemm_kernel_base.cpp

```diff
@@ -830,7 +830,8 @@
 
 namespace rocwmma
 {
-    bool KernelI::sHeaderPrinted = false;
+    bool KernelI::sHeaderPrinted  = false;
+    bool KernelI::sFHeaderPrinted = false;
```

Comment:
Don't need knowledge here about whether the header has been printed in file output. See below